File: app/src/main/cs/Challenge0.cs
using System;

namespace FizzBuzzer;

public class Program
{
    private static void Main(string[] args)
    {
        Console.WriteLine("\t\t* * Ejercicio #0 - FIZZ BUZZ * *");

        for (int number = 1; number <= 100; number++)
        {
            bool isMultipleOf3 = number % 3 == 0;
            bool isMultipleOf5 = number % 5 == 0;

            if (isMultipleOf3 && isMultipleOf5)
            {
                Console.Write("fizzbuzz");
            }
            else if (isMultipleOf3)
            {
                Console.Write("fizz");
            }
            else if (isMultipleOf5)
            {
                Console.Write("buzz");
            }
            else
            {
                Console.Write(number);
            }

            Console.Write("\n");
        }
    }
}

File: app/src/main/java/com/mouredev/weeklychallenge2022/Challenge0.go
package main

import "fmt"

func main() {
	i := 1
	for i <= 100 {
		byThree := i%3 == 0
		byFive := i%5 == 0

		if byThree && byFive {
			fmt.Printf("%d FIZZ BUZZ\n", i)
		} else if byFive {
			fmt.Printf("%d BUZZ\n", i)
		} else if byThree {
			fmt.Printf("%d FIZZ\n", i)
		} else {
			fmt.Println(i)
		}

		i++
	}
}
